Packages/com.unity.inputsystem/InputSystem/Plugins/Android/AndroidSensors.cs

```diff
@@ -257,6 +259,15 @@ public class AndroidAmbientTemperature : AmbientTemperatureSensor
     {
     }
 
+    /// <summary>
+    /// Game rotation vector sensor device on Android.
+    /// </summary>
+    /// <seealso href="https://developer.android.com/reference/android/hardware/Sensor#TYPE_GAME_ROTATION_VECTOR"/>
+    [InputControlLayout(stateType = typeof(AndroidSensorState), variants = "GameRotationVector", hideInUI = true)]
+    public class AndroidGameRotationVector : AttitudeSensor
```

Comment:
Looks good, wondering if we should mention something in https://docs.unity3d.com/Packages/com.unity.inputsystem@1.0/manual/Sensors.html#attitudesensor for Android regarding that it either picks RotationVector or GameRotationVector sensor, cc 
@jamesmcgill

Reply:
I'm really not sure. I don't think adding more documentation could hurt though, so if you think it might be useful for a user to know then why not.